job_spec/RIVER_WIDTH.yml

```
@@ -43,7 +43,19 @@ RIVER_WIDTH:
         - Ref::granules
       timeout: 36000
       vcpu: 1
-      memory: 126000
+      memory: 31600
+    - name: WATER_MAP
+      image: ghcr.io/asfhyp3/asf-tools
+      command:
+        - ++process
+        - water_map
+        - --bucket
+        - '!Ref Bucket'
+        - --bucket-prefix
+        - Ref::bucket_prefix
+      timeout: TODO
+      vcpu: TODO
+      memory: TODO
```

```suggestion
      vcpu: 1
      memory: 126000
```
looks like we upped the memory for water map jobs int he river width spec, so I suspect asf-tools needs a lot of memory for 10m water maps.